ng-dev/pr/merge/pull-request.ts

```diff
@@ -290,3 +273,32 @@ function assertPendingState(pr: RawPullRequest) {
       throw PullRequestFailure.isMerged();
   }
 }
+
+/** Get the branches the pull request will be merged into.  */
+async function getTargetBranches(
+  config: {merge: MergeConfig; github: GithubConfig},
+  labels: string[],
+  targetBranch: string,
```

Comment:
We should keep the `githubTargetBranch` terminology to avoid ambiguity with actual target branches being determined. it's obviously reasonable from the context, but doesn't hurt staying more explicit.

```suggestion
  githubTargetBranch: string,
```